src/Umbraco.Web.UI.Client/src/packages/tiptap/components/character-map/character-map-modal.token.ts

```diff
@@ -0,0 +1,12 @@
+import { UmbModalToken } from '@umbraco-cms/backoffice/modal';
+
+export type UmbCharacterMapModalData = never;
+
+export type UmbCharacterMapModalValue = string;
+
+export const UMB_CHARACTER_MAP_MODAL = new UmbModalToken<UmbCharacterMapModalData, UmbCharacterMapModalValue>(
+	'Umb.Modal.CharacterMap',
```

Comment:
Should this be defined in a `constants.ts` file and exported separately, or what is the convention nowadays?

The name is generic, but the whole modal is defined within the tiptap package. Can it be used outside of tiptap? If yes, maybe consider moving it, and if no, consider being more specific?